CHANGES.rst

```diff
@@ -103,6 +103,11 @@ Internal Changes
   pre-filled when the user then opens the link returned by that endpoint (:pr:`4628`, thanks
   :user:`adl1995`)
 
+Internationalization
+^^^^^^^^^^^^^^^^^^^^
+
+- New translation: Turkish
```

Comment:
Can you put it before "Improvements"? Just to keep a consistent order of the changelog categories..

Reply:
Yes, my bad, I didn't notice it